arkouda/groupbyclass.py

```diff
@@ -530,6 +541,135 @@ def prod(self, values: pdarray, skipna: bool = True) -> Tuple[groupable, pdarray
         k, v = self.aggregate(values, "prod", skipna)
         return k, cast(pdarray, v)
 
+    def var(
+        self, values: pdarray, skipna: bool = True, ddof: int_scalars = 1
+    ) -> Tuple[groupable, pdarray]:
+        """
+        Using the permutation stored in the GroupBy instance, group
+        another array of values and compute the variance of
+        each group's values.
+
+        Parameters
+        ----------
+        values : pdarray
+            The values to group and find variance
+        skipna: bool
+            boolean which determines if NANs should be skipped
+        ddof : int_scalars
+            "Delta Degrees of Freedom" used in calculating var
+
+        Returns
+        -------
+        unique_keys : (list of) pdarray or Strings
+            The unique keys, in grouped order
+        group_vars : pdarray, float64
+            One var value per unique key in the GroupBy instance
+
+        Raises
+        ------
+        TypeError
+            Raised if the values array is not a pdarray object
+        ValueError
+            Raised if the key array size does not match the values size
+            or if the operator is not in the GroupBy.Reductions array
+
+        Notes
+        -----
+        The return dtype is always float64.
+
+        The variance is the average of the squared deviations from the mean,
+        i.e.,  ``var = mean((x - x.mean())**2)``.
+
+        The mean is normally calculated as ``x.sum() / N``, where ``N = len(x)``.
+        If, however, `ddof` is specified, the divisor ``N - ddof`` is used
+        instead.  In standard statistical practice, ``ddof=1`` provides an
+        unbiased estimator of the variance of a hypothetical infinite population.
+        ``ddof=0`` provides a maximum likelihood estimate of the variance for
+        normally distributed variables.
+
+        Examples
+        --------
+        >>> a = ak.randint(1,5,10)
+        >>> a
+        array([3, 3, 4, 3, 3, 2, 3, 2, 4, 2])
+        >>> g = ak.GroupBy(a)
+        >>> g.keys
+        array([3, 3, 4, 3, 3, 2, 3, 2, 4, 2])
+        >>> b = ak.randint(1,5,10)
+        >>> b
+        array([3, 3, 3, 4, 1, 1, 3, 3, 3, 4])
+        >>> g.mean(b)
```

Comment:
I would expect this example to call `g.var()` instead of `g.mean()`

Reply:
good catch!!